linkml/generators/csvgen.py

```diff
@@ -55,7 +55,7 @@ def visit_class(self, cls: ClassDefinition) -> bool:
 
 @shared_arguments(CsvGenerator)
 @click.command()
-@click.option("--root", "-r", default=None, multiple=True, help="Class(es) to transform")
+@click.option("--root", "-r", default=None, show_default=True, multiple=True, help="Class(es) to transform")
```

Comment:
is it useful to do this is default is none?

Reply:
Oh, that's a good point. Probably not. I think we should remove `show_default=True` when `default=None`. Also I don't think we need to explicitly specify `default=None`, I think the option value is `None` implicitly.

Reply:
oh you are quite right, I misread!!